core/src/main/java/com/valtech/aemsaas/core/services/configuration/BaseHttpServiceConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package com.valtech.aemsaas.core.services.configuration;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+import static com.valtech.aemsaas.core.util.Constants.*;
+
+
+@ObjectClassDefinition(name = "SAAS 01 BaseSearchService", description = "URL and credentials to connect to Search as a Service (SAAS).")
```

Comment:
remove the number please and rename to
`Search as a Service - HTTP Service Configuration`

- [ ] Please rename the class as well and remove the "Base" . It doesn't provide any additional context